### DISCUSSION TITLE
new OS and genmon install on old pi crashes every day around 2am

### DISCUSSION THREAD

**Discussion by mac1-131** [jgyates/genmon#1127] [Questions] [answered]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open issues](https://github.com/jgyates/genmon/issues),[closed issues](https://github.com/jgyates/genmon/issues?q=is%3Aissue+is%3Aclosed),[open discussion](https://github.com/jgyates/genmon/discussions), [closed discussion](https://github.com/jgyates/genmon/discussions/categories/questions?discussions_q=is%3Aclosed) - which may answer my question or that I could contribute my information to.
- [X] I have read the [Known Issues Wiki Page](https://github.com/jgyates/genmon/wiki/Appendix-D-Known-Issues) and my question / issue is not listed.
- [X] I have provided the version of the software and the operating system in my post.
- [X] If I have an issue with genmon not communicating with the controller, I have read and followed the [serial troubleshooting guide](https://github.com/jgyates/genmon/wiki/3.6---Serial-Troubleshooting).

### Version

1.19.03

### Have you submitted your log files from the About page? Note that outbound email must be working for logs to be submitted.

I have submitted logs

### What is/are your question(s)?

I submitted some logs this morning. I had an existing working genmon but I wanted to get rustdesk working on it and the only pi OS I could get rustdesk to install on was 11 (bullseye) which was an update from what I was running (9) so I installed that and put genmon on it without incident.

It works just fine, but every night at around 2am it crashes with some kernel log entries about python3 oom_killer. Up until this 2am(ish) crash, genmon works perfectly. Seems about 12 hours after initial boot.

Certainly not sure if this is genmon related or not (probably not) but looking for any ideas

it is a pi 3b, worked fine under OS 9 and genmon 1.19.00

I did see something in there about swap file size of 0 - I have it set to 512, old one was 1024 so I will make that change and see what happens tonight. pi 3b only has 1gb memory so not sure if making swap 1024 is OK or not. I had the old one that way, so will try it

going away for a couple weeks so if I can't get this under control in the next day or so will just put my old sd card back in

any suggestions appreciated, thanks.

**Comment by jgyates** [comment]
did you upgrade from 9 to 11 or do a complete reinstall? Generally upgraidgin from one version of linux to another is a crap shoot. It may work but have strange problems. 

If you did a fresh install then you may have some bad spots on your SD card. Using a new card may be the trick. This is just a guess.

**Comment by mac1-131** [comment, marked as answer]
OK pretty sure this is not genmon related. I had clamav on this pi and it is pretty sketchy running that on a pi 3b. I increated swap to 1024 and removed clamav and it survived the night OK.
